README.md

```diff
@@ -7,6 +7,106 @@
 
 This is a desktop application acting as a Gradle Tooling API client.
 
+## Usage
+
+You can download the latest version of _Gradle Client_ for macos, Linux or Windows from the [Releases](https://github.com/gradle/gradle-client/releases) assets.
+
+Once installed on your system, simply launch the application.
```

Comment:
💭 Installed?  This makes it sound like a much more heavyweight application.  This is just download and run.

Reply:
The binaries are respectively `dmg`, `deb` and `msi` files, so they need to be installed.

Reply:
Oh, ok.  I just built and ran the project and didn't notice that.